src/components/BreedsList.vue

```diff
@@ -168,24 +165,15 @@ export default {
       }
     },
     toggleCard(breed) {
-      console.log(breed.isActive, breed.name, this.$route.name);
       breed.isActive = !breed.isActive;
       this.displayedBreeds.forEach((item) => {
         if (item.name !== breed.name) {
           item.isActive = false;
         }
       });
-      if (!breed.isActive) {
-        console.log("push");
-        this.$router.push({
-          name: "breeds",
-        });
-      }
```

Comment:
Bug, ki se je pojavil , sem rešil s tem, da sem umaknil to kodo, ta koda pusha route na `breeds`, ampak ko odpiram in zapiram kartico je vedno isti route breeds , torej ni potrebe po tej zgornji kodi, kar samo pusha na isti route, zato tudi ta napaka